### ISSUE TITLE
comment-format should ignore //@ts-...

### ISSUE THREAD

**Issue by KSXGitHub** [palantir/tslint#4194]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 3.1.0
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
//@ts-ignore
```

with `tslint.json` configuration:

```json
{
  "comment-format": [true, "check-space"]
}
```

#### Actual behavior

`//@ts-...` is a TypeScript specific instructions that only work without space, TSLint should recognize them.

#### Expected behavior

TSLint treated them like normal comments.

**Comment by ajcrites** [comment]
`// @ts-` will still work though

**Comment by KSXGitHub** [comment]
You are right! I should've double checked that. Sorry.
